modules/odc-exporting-an-application-to-another-project-or-cluster.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="odc-exporting-an-application-to-another-project-or-cluster_{context}"]
+
+= Exporting an application to another project or cluster
+
+As a developer, you can export your application in the ZIP file format. Based on your needs, import the exported application to another project in the same cluster or a different cluster by using the *Import YAML* option in the *+Add* view. This helps you to reuse your application resources and saves your time.
+
+.Prerequisites
+You have installed the gitops-primer operator from the Operator Hub.
```

Comment:
In the topology view, to enable the `Export application` button an application needs to be created. Can you add this point as well?

Reply:
Incorporated this point @debsmita1